procServUtils/launch.py

```diff
@@ -7,7 +7,7 @@
 except ImportError:
     from . import shlex
 
-procServ = '/usr/bin/procServ'
+procServ = '/usr/local/bin/procServ'
```

Comment:
Probably should search `$PATH` for executable as the location will depend on how procServ is installed (RPM vis sudo make install).